Nataly_Bondarenko/test_framework/utilities/driver_factory.py

```diff
@@ -0,0 +1,23 @@
+from selenium.webdriver import Chrome, Firefox, Edge
+from selenium.webdriver.chrome.service import Service
```

Comment:
You import Service for chrome only. But you have 3 different browsers here

Reply:
fixed